_config/acc_staging.yml

```diff
@@ -1,4 +1,4 @@
-id: acc_staging
+id: acc
```

Comment:
Why is this changing? This ID value dictates much of what happens in the rake tasks to support the environment-specific configurations.